server/election/lease.go

```diff
@@ -83,17 +86,17 @@ func (l *lease) Close() error {
 // IsExpired checks if the lease is expired. If it returns true,
 // current leader should step down and try to re-elect again.
 func (l *lease) IsExpired() bool {
-	if l == nil {
+	if l == nil || l.expireTime.Load() == nil {
 		return true
 	}
-	if l.expireTime.Load() == nil {
-		return false
-	}
```

Comment:
Involved in https://github.com/tikv/pd/commit/7a6a9daaf37e0cf9e444ccc613a6c70bcccdd05c#diff-ce11c8a270a68ea01352751b710c90d380002a3a037cbd39e588e963e24863e0R81, which is not correct. Fixed by this PR.